cve_bin_tool/util.py

```diff
@@ -7,37 +7,41 @@
 import sys
 from collections import defaultdict
 from enum import Enum
-from typing import NamedTuple
+from typing import List, NamedTuple, Optional, Set, Tuple, Union
 
 
 class OrderedEnum(Enum):
-    def __ge__(self, other):
+    def __ge__(self, other) -> bool:
         if self.__class__ is other.__class__:
             return self.value >= other.value
         return NotImplemented
 
-    def __gt__(self, other):
+    def __gt__(self, other) -> bool:
         if self.__class__ is other.__class__:
             return self.value > other.value
         return NotImplemented
 
-    def __le__(self, other):
+    def __le__(self, other) -> bool:
         if self.__class__ is other.__class__:
             return self.value <= other.value
         return NotImplemented
 
-    def __lt__(self, other):
+    def __lt__(self, other) -> bool:
         if self.__class__ is other.__class__:
             return self.value < other.value
         return NotImplemented
 
 
+tup_4 = Tuple[int, str, str, str, str]
+tup_5 = Tuple[int, str, str, str, str, str]
```

Comment:
Hm, what type checker do you use? I actually get more warnings because of these tuples:
```
Expression of type "tuple[Literal[1], Literal['1'], Literal['NewFound'], Literal['n'], Literal['N']]" cannot be assigned to declared type "Literal[Remarks.NewFound]"
  "tuple[Literal[1], Literal['1'], Literal['NewFound'], Literal['n'], Literal['N']]" cannot be assigned to type "Literal[Remarks.NewFound]"
```

I think we can delete these.

Reply:
I use mypy, and didn't get these warnings.
Anyway, I will delete them, so just leave the tuples as is, without annotations?

Reply:
On a second note, I could not find much about type hinting enum members like this.